.github/workflows/smoke-checks.yml

```diff
@@ -168,23 +168,19 @@ jobs:
     name: Build LLC + UI (Xcode 12)
     runs-on: macos-11
     if: ${{ github.event_name != 'push' }}
+    env:
+      XCODE_VERSION: "12.5.1"
```

Comment:
This has been moved to 13, and temporarily disabled until we move to Xcode 14, FYI